ktor-network/nix/test/io/ktor/network/sockets/tests/SelectNixTest.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.sockets.tests
+
+import io.ktor.network.selector.*
+import io.ktor.network.selector.EventInfo
+import io.ktor.network.selector.SelectorHelper
+import io.ktor.test.dispatcher.*
+import kotlinx.coroutines.*
+import kotlin.coroutines.*
+import kotlin.test.*
+
+class SelectNixTest {
+    @Test
+    @Ignore
```

Comment:
It's strange to ignore the test in the new PR without an issue, could you tell me why?